samples/AspNetWebApp/ScenariosController.cs

```diff
@@ -0,0 +1,50 @@
+using System.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Client;
+
+namespace AspNetWebApp;
+
+[Route("scenarios")]
+[ApiController]
+public partial class ScenariosController(
+    DurableTaskClient durableTaskClient,
+    ILogger<ScenariosController> logger) : ControllerBase
+{
+    readonly DurableTaskClient durableTaskClient = durableTaskClient;
+    readonly ILogger<ScenariosController> logger = logger;
+
+    [HttpPost("hellocities")]
+    public async Task<ActionResult> RunHelloCities([FromQuery] int? count, [FromQuery] string? prefix)
+    {
+        if (count is null || count < 1)
+        {
+            return this.BadRequest(new { error = "A 'count' query string parameter is required and it must contain a positive number." });
+        }
+
+        // Generate a semi-unique prefix for the instance IDs to simplify tracking
+        prefix ??= $"hellocities-{count}-";
+        prefix += DateTime.UtcNow.ToString("yyyyMMdd-hhmmss");
+
+        this.logger.LogInformation("Scheduling {count} orchestrations with a prefix of '{prefix}'...", count, prefix);
+
+        Stopwatch sw = Stopwatch.StartNew();
```

Comment:
Recommend removing the stopwatch log. This is redundant with distributed tracing, which will already calculate the duration for this API call.

If you really want to have a sample showing duration to enqueue orchestrations, I recommend looking into custom `ActivitySource` and using an internal activity (distributed tracing again)

Reply:
I don't think this is a good sample for us to include in the public repo. The original code was not designed to be a sample, but rather to be a performance testing benchmark tool. I suggest we do something simpler for this PR.

Reply:
deleted